prow/config/config.yaml

```diff
@@ -365,6 +365,22 @@ presubmits:
         imagePullPolicy: Always
 
   metal3-io/cluster-api-provider-metal3:
+  - name: gomod
+    branches:
+    - main
```

Comment:
can you please add "release-1.1" to this list?

Reply:
I haven’t added the gomod file in that branch yet.

Reply:
yes, but we can add that branch with this PR and backport the https://github.com/metal3-io/cluster-api-provider-metal3/pull/725 to release-1.1 branch once merged

Reply:
Done